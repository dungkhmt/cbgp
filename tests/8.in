50 50
50 80
14 3
49 42
36 15
19 11
27 6
47 17
47 2
21 5
28 7
34 29
32 26
37 4
20 5
29 14
4 5
32 13
43 0
6 22
49 36
18 20
43 10
32 10
6 2
24 0
41 6
28 1
36 41
48 24
32 25
45 4
30 9
20 23
38 6
24 7
35 45
49 39
9 4
9 12
15 0
25 9
5 8
26 11
3 0
40 15
34 16
46 39
44 26
32 33
31 9
47 7
37 45
27 28
32 11
29 3
24 2
2 3
35 37
34 14
32 9
27 1
22 14
3 16
13 10
42 18
18 5
17 7
42 5
11 16
42 15
12 0
43 19
30 23
27 36
36 1
26 4
32 31
11 3
43 33
18 23
6 3
1
30 5 71 2 60 3 57 6 29 21 4 4 63 11 66 2 22 22 38 13 36 8 48 7 36 11 37 9 52 11 26 1 49 8 64 3 18 5 37 7 15 6 0 0 57 16 16 7 33 2 39 11 31 23 70 4 68 2 53 9 29 20 39 14 45 8 37 10 50 9 25 22 76 0 27 22 42 32 4 0 28 2 64 12 22 1 34 7 40 32 25 25 2 0 63 5 30 2 21 0