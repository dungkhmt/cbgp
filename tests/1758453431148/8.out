(14, 3), (49, 42), (36, 15), (19, 11), (27, 6), (47, 17), (47, 2), (21, 5), (28, 7), (34, 29), (32, 26), (37, 4), (20, 5), (29, 14), (4, 5), (32, 13), (43, 0), (6, 22), (49, 36), (18, 20), (43, 10), (32, 10), (6, 2), (24, 0), (41, 6), (28, 1), (36, 41), (48, 24), (32, 25), (45, 4), (30, 9), (20, 23), (38, 6), (24, 7), (35, 45), (49, 39), (9, 4), (9, 12), (15, 0), (25, 9), (5, 8), (26, 11), (3, 0), (40, 15), (34, 16), (46, 39), (44, 26), (32, 33), (31, 9), (47, 7), (37, 45), (27, 28), (32, 11), (29, 3), (24, 2), (2, 3), (35, 37), (34, 14), (32, 9), (27, 1), (22, 14), (3, 16), (13, 10), (42, 18), (18, 5), (17, 7), (42, 5), (11, 16), (42, 15), (12, 0), (43, 19), (30, 23), (27, 36), (36, 1), (26, 4), (32, 31), (11, 3), (43, 33), (18, 23), (6, 3), 
numberIntersection=0 
minDistance=2.197255145531258 
minAngle=0.02940328820400495 
sumAngle=56.39151269481982 
sumDistance=256215.93861524816 
minEdgeLength=2.8284271247461903 
sumEdgeLength=2412.1181850824573 

0.0000000000 2.1972551455 0.0294032882 56.3915126948 256215.9386152482 2.8284271247 2412.1181850825
0: (21, 144),
1: (3, 53),
2: (23, 83),
3: (25, 85),
4: (78, 140),
5: (8, 195),
6: (45, 69),
7: (3, 62),
8: (151, 200),
9: (57, 126),
10: (30, 127),
11: (26, 103),
12: (35, 137),
13: (32, 123),
14: (45, 91),
15: (3, 149),
16: (29, 100),
17: (6, 67),
18: (17, 156),
19: (27, 119),
20: (24, 170),
21: (1, 200),
22: (45, 72),
23: (35, 170),
24: (5, 116),
25: (41, 118),
26: (87, 107),
27: (7, 53),
28: (3, 56),
29: (34, 94),
30: (55, 130),
31: (48, 117),
32: (29, 107),
33: (35, 124),
34: (33, 101),
35: (157, 198),
36: (0, 47),
37: (100, 196),
38: (195, 0),
39: (0, 195),
40: (6, 145),
41: (37, 63),
42: (3, 157),
43: (28, 134),
44: (200, 0),
45: (154, 136),
46: (0, 198),
47: (8, 65),
48: (5, 142),
49: (0, 157),
21 144 3 53 23 83 25 85 78 140 8 195 45 69 3 62 151 200 57 126 30 127 26 103 35 137 32 123 45 91 3 149 29 100 6 67 17 156 27 119 24 170 1 200 45 72 35 170 5 116 41 118 87 107 7 53 3 56 34 94 55 130 48 117 29 107 35 124 33 101 157 198 0 47 100 196 195 0 0 195 6 145 37 63 3 157 28 134 200 0 154 136 0 198 8 65 5 142 0 157 
0: (21, 144), 1: (3, 53), 2: (23, 83), 3: (25, 85), 4: (78, 140), 5: (8, 195), 6: (45, 69), 7: (3, 62), 8: (151, 200), 9: (57, 126), 10: (30, 127), 11: (26, 103), 12: (35, 137), 13: (32, 123), 14: (45, 91), 15: (3, 149), 16: (29, 100), 17: (6, 67), 18: (17, 156), 19: (27, 119), 20: (24, 170), 21: (1, 200), 22: (45, 72), 23: (35, 170), 24: (5, 116), 25: (41, 118), 26: (87, 107), 27: (7, 53), 28: (3, 56), 29: (34, 94), 30: (55, 130), 31: (48, 117), 32: (29, 107), 33: (35, 124), 34: (33, 101), 35: (157, 198), 36: (0, 47), 37: (100, 196), 38: (195, 0), 39: (0, 195), 40: (6, 145), 41: (37, 63), 42: (3, 157), 43: (28, 134), 44: (200, 0), 45: (154, 136), 46: (0, 198), 47: (8, 65), 48: (5, 142), 49: (0, 157), 
numberIntersection=0 minAngle=0.02940328820400495 sumAngle=56.391512694631224 minEdgeLength=2.8284271247461903 sumEdgeLength=2412.11818508247 minDistance=2.197255145531258 sumDistance=256215.93861522595 
Time taken: 40.408 s
